README.md

```diff
@@ -114,6 +110,45 @@ All the following attributes correspond to the ones listed in the
 * [on-upload-complete](https://uploadcare.com/docs/api_reference/javascript/widget/#widget-on-upload-complete)
 * [on-change](https://uploadcare.com/docs/api_reference/javascript/widget/#widget-on-change)
 
+Events usage example:
+```javascript
+  onUpload(info) {
+    console.log('fired Event "onUpload"');
+    console.log(info);
+  }
+  onChange(info) {
```

Comment:
use the `file` name instead of `info`

https://uploadcare.com/docs/api_reference/javascript/widget/#widget-on-change